dali/test/python/test_operator_multipaste.py

```diff
@@ -220,19 +224,34 @@ def test_operator_multipaste():
-        [4, 2, (128, 128), (256, 128), True, False, False, False, False, types.UINT8],
-        [4, 2, (128, 128), (128, 256), True, True, False, False, False, types.UINT8],
-
-        [4, 2, (64, 64), (128, 128), False, False, True, False, False, types.UINT8],
-        [4, 2, (64, 64), (128, 128), False, False, False, True, False, types.UINT8],
-        [4, 2, (64, 64), (128, 128), False, False, False, False, True, types.UINT8],
-
-        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.UINT8],
-        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.INT16],
-        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.INT32],
-        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.FLOAT]
+        # - should use GPU operator
+        [4, 2, (128, 256), (128, 128), False, False, False, False, False, types.UINT8, False],
+        [4, 2, (256, 128), (128, 128), False, True, False, False, False, types.UINT8, False],
+        [4, 2, (128, 128), (256, 128), True, False, False, False, False, types.UINT8, False],
+        [4, 2, (128, 128), (128, 256), True, True, False, False, False, types.UINT8, False],
+
+        [4, 2, (64, 64), (128, 128), False, False, True, False, False, types.UINT8, False],
+        [4, 2, (64, 64), (128, 128), False, False, False, True, False, types.UINT8, False],
+        [4, 2, (64, 64), (128, 128), False, False, False, False, True, types.UINT8, False],
+
+        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.UINT8, False],
+        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.INT16, False],
+        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.INT32, False],
+        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.FLOAT, False],
+
+        [4, 2, (128, 256), (128, 128), False, False, False, False, False, types.UINT8, True],
+        [4, 2, (256, 128), (128, 128), False, True, False, False, False, types.UINT8, True],
+        [4, 2, (128, 128), (256, 128), True, False, False, False, False, types.UINT8, True],
+        [4, 2, (128, 128), (128, 256), True, True, False, False, False, types.UINT8, True],
+
+        [4, 2, (64, 64), (128, 128), False, False, True, False, False, types.UINT8, True],
+        [4, 2, (64, 64), (128, 128), False, False, False, True, False, types.UINT8, True],
+        [4, 2, (64, 64), (128, 128), False, False, False, False, True, types.UINT8, True],
+
+        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.UINT8, True],
+        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.INT16, True],
+        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.INT32, True],
+        [4, 2, (128, 128), (128, 128), False, False, False, False, False, types.FLOAT, True]
     ]
     for t in tests:
         yield (check_operator_multipaste, *t)
```

Comment:
Instead of copy-pasting the table you could do:
```suggestion
        yield (check_operator_multipaste, *t, "cpu")
        yield (check_operator_multipaste, *t, "gpu")
```